dart/biomechanics/DynamicsFitter.cpp

```diff
@@ -10143,6 +10153,7 @@ void DynamicsFitter::estimateUnmeasuredExternalForces(
                   << trial << " at time " << t << std::endl;
         filteredTimesteps.push_back(t + 1);
         init->probablyMissingGRF[trial][t + 1] = true;
+        init->missingGRFReason[trial][t + 1] = MissingGRFReason::unmeasuredForce;
```

Comment:
Maybe we rename this to `unmeasuredExternalForceDetected` or something, to make it a bit clearer what's happening?